src/TestUtils/src/UITest.Appium/Actions/AppiumSwipeActions.cs

```diff
@@ -91,9 +91,9 @@ static void SwipeToRight(AppiumDriver driver, AppiumElement? element, double swi
 			int endY = startY;
 
 			new TouchAction(driver)
-				.Press(startX, startY)
-				.Wait(swipeSpeed)
+				.Press(startX, startY)	
 				.MoveTo(endX, endY)
+				.Wait(swipeSpeed)
```

Comment:
Changes in the order of the actions to avoid a `DriverException:  Method has not yet been implemented`.